app/assets/stylesheets/thredded/components/_messageboard.scss

```diff
@@ -94,6 +94,10 @@
   margin-bottom: 0;
 }
 
+.thredded--messageboard--delete {
+  background-color: red;
```

Comment:
Use `$thredded-alert-danger-color` instead of `red`.

All the colors in Thredded are fully customizable through variables defined in https://github.com/thredded/thredded/blob/master/app/assets/stylesheets/thredded/base/_variables.scss.

Reply:
Done.